notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>...and more generally N manifolds M1, ..., M_N can be combined to create a new manifold M.</p><p> </p><p>For example, we can create a product of two spheres as:</p><pre class="ql-syntax" spellcheck="false">from geomstats.geometry.product_manifold import ProductManifold

sphere1 = Hypersphere(dim=2)
sphere2 = Hypersphere(dim=2)

product_of_two_spheres = ProductManifold([sphere1, sphere2])
</pre><p> </p><p>If we generate a random point on this product manifold, we will get 6 coordinates: the first 3 represent a point on the first sphere, and the last 3 represent a point on the second sphere:</p><pre class="ql-syntax" spellcheck="false">product_of_two_spheres.random_point()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>made this change!</p>